test/cypress/e2e/delegateReg.spec.js

```diff
@@ -31,41 +18,60 @@ describe('Delegate Registration', () => {
     randomDelegateName = getRandomDelegateName();
   });
 
-  it(`opens by url ${urls.registerDelegate}`, () => {
+  /**
+   * Delegate registration page can be opened by direct link
+   * @expect url is correct
+   * @expect some specific to page element is present on it
+   */
+  it(`Opens by url ${urls.registerDelegate}`, () => {
     cy.autologin(accounts['delegate candidate'].passphrase, networks.devnet.node);
     cy.visit(urls.registerDelegate);
     cy.url().should('contain', urls.registerDelegate);
-    cy.get(ss.chooseName);
+    cy.get(ss.chooseDelegateName);
   });
-  // TODO Add balance substraction check after #1391 is fixed
-  it('register delegate', () => {
+
+  /**
+   * Go through register delegate process
+   * @expect transaction appears in the activity list in the confirmed state with valid details
+   * @expect header balance value is decreased
+   */
+  it('Register delegate + Header balance is affected', function () {
```

Comment:
can you please mention what are the steps of the process instead of process?

Reply:
I agree. The steps can be described either in the comment above or inside the `it`